File: cpp/level 1/Find a decimal number.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    if(n==2)
        return 1;
    for(int i=2;i<n;i++) {
		if(n % i == 0) {
			return 0 + solution(n-1);
			break;
		}
	}
    return 1 + solution(n-1);
}
/**/

File: Python/level 1/Find a decimal number.py
def solution(n):
    arr = [i for i in range(n+1)]
    arr[0],arr[1] = 0,0
    for i in range(2,n+1):
        if arr[i] != 0:
            if i*i > 1000001:
                break
            else:
                for j in range((i*i),n+1,i):
                    arr[j] = 0
    return len(arr)-arr.count(0)